pkg/agent/proxy/proxier.go

```diff
@@ -50,6 +52,7 @@ const (
 	// SessionAffinity timeout is implemented using a hard_timeout in OVS. hard_timeout is
 	// represented by a uint16 in the OpenFlow protocol,
 	maxSupportedAffinityTimeout = math.MaxUint16
+	mcServicePrefix             = "antrea-mc-"
```

Comment:
Name it `mcServiceNamePrefix` or add some comments to explain it is the prefix added to the MC Service name.

Reply:
Done